main.js

```diff
@@ -18,6 +18,12 @@ async function main() {
     core.setOutput('number', pr && pr.number || '');
     core.setOutput('title', pr && pr.title || '');
     core.setOutput('body', pr && pr.body || '');
+    if (pr && pr.labels) {
+        core.setOutput('labels', pr.labels.map(label => label.name));
```

Comment:
I'm not sure you're allowed to set an output to a complex type like this; I think they just have to be strings?

Reply:
You might be right, I just read through this document:
and assumed that they could be set to anything...
I still didn't manage to see the result of this, I'll update once I managed to get it working with a test PR...

Reply:
I got it working using the approach from this fork:
https://github.com/mhretab/gh-find-current-pr (it directly grabs the data from `context.payload`), since I'm not sure if this covers all cases/how to test that, I'll keep those changes in our fork. 

The `labels` value seem to have been stored without problems, but I have not attempted to use it. (Will give it a try just to be sure).

The `label_...` approach worked perfectly, but those keys are of course dynamic and can not be defined in the `action.yml`...

Reply:
> and assumed that they could be set to anything...

It might well be that it can now.  I tried stuff like this in the early days of GH Actions and it didn't work for me.  :P  But if it works, that's awesome.